blogposts/2021/release-3.31.md

```diff
@@ -0,0 +1,77 @@
+    url: https://github.com/sourcegraph/sourcegraph/pull/23736
+    category: Search
+  - description: Updating draft merge requests on GitLab from batch changes no longer removes the draft status.
+    url: https://github.com/sourcegraph/sourcegraph/issues/23944
+    category: Batch Changes
+---
+
+Sourcegraph 3.31 is now available! For this release, we introduced:
+
+## Code Insights now in Private Beta
+
+Code Insights enables you to track historical and future trends of anything in your codebase, from code migrations to API usage to linter overrides—or anything else you can run a Sourcegraph search for. Now in Private Beta, Code Insights can run across thousands of repositories, letting you then filter and explore your insights live. Reach out to your Account Executive for details.
+
+<div style="text-align:center"><video autoplay loop muted playsinline style="width:625px">
+  <source src="https://sourcegraphstatic.com/blog/3.31/code_insights_3.31_beta.mp4" type="video/mp4">
+</video></div>
+
+- Code Insights runs over all your repositories—we've removed the ~50-70 repo limit.
+- [Instant filtering](https://docs.sourcegraph.com/code_insights/explanations/code_insights_filters) allows you to explore insights using `repo:` and `-repo:` regular expressions.
+- [Save your filtered insights as new views](https://docs.sourcegraph.com/code_insights/explanations/code_insights_filters#filter-persistance-and-sharing).
+
+## We're moving to Alpine-based database images
+
+We've switched our built-in main Postgres and codeintel databases to an Alpine-based Docker image to resolve vulnerabilities found in Debian. This requires re-indexing the database when upgrading to 3.31, which can take up to a few hours on systems with large datasets. [Read more on how to prepare for the update here](https://docs.sourcegraph.com/admin/migration/3_31).
+
+## Saved search notifications are migrating to code monitoring
+
+Email notifications for saved searches are being deprecated in favor of code monitoring. Like email notifications for saved searches, code monitoring allows you to keep track of and get notified about changes in your code. Some use cases for code monitors include getting notifications for potential secrets, anti-patterns, or common typos committed to your codebase. Read more about [code monitoring](https://docs.sourcegraph.com/code_monitoring).
+
+If you are using email notifications for existing saved searches, there is now a button to automatically migrate to code monitoring. Note that existing email notifications for saved searches will be removed in the future.
+
+## We're now indexing the open source universe on Sourcegraph cloud
+
+Sourcegraph is indexing every open source GitHub and GitLab repository with 1 or more stars. Users can search this index of open source code on [Sourcegraph cloud](https://www.sourcegraph.com), and you can read about [why we're doing this on our blog](https://about.sourcegraph.com/blog/why-index-the-oss-universe/).
```

Comment:
This implies we have all these repositories now, not that we're working on doing it. 

is indexing -> is working to index instead maybe?

Reply:
Good point— I'll update the subhead here too